azurerm/internal/services/logic/tests/logic_app_trigger_recurrence_resource_test.go

```diff
@@ -206,6 +206,27 @@ func TestAccAzureRMLogicAppTriggerRecurrence_startTime(t *testing.T) {
 	})
 }
 
+func TestAccAzureRMLogicAppTriggerRecurrence_startTimeWithTimeZone(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_logic_app_trigger_recurrence", "test")
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acceptance.PreCheck(t) },
+		Providers:    acceptance.SupportedProviders,
+		CheckDestroy: testCheckAzureRMLogicAppWorkflowDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAzureRMLogicAppTriggerRecurrence_startTimeWithTimeZone(data, "2020-01-01T01:02:03Z", "U.S. Eastern Standard Time"),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMLogicAppTriggerExists(data.ResourceName),
+					resource.TestCheckResourceAttr(data.ResourceName, "start_time", "2020-01-01T01:02:03Z"),
+					resource.TestCheckResourceAttr(data.ResourceName, "time_zone", "U.S. Eastern Standard Time"),
+				),
+			},
+			data.ImportStep(),
```

Comment:
since it's possible to update this field could we also include a second configuration/import step here to ensure this can be updated?